js/src/personal/haoking/Haoking.page.tsx

```diff
@@ -1,24 +1,39 @@
-import { MantineProvider, Box, Flex, rem } from '@mantine/core'
-import { LeftAboutMe } from './LeftAboutMe.tsx'
-import { RightAboutMe } from './RightAboutMe.tsx'
-import { theme } from './theme.ts'
+import { useState } from 'react'
+import { HaokingAboutMe } from './Haoking.aboutme.tsx'
+import { HaokingProjects } from './Haoking.projects.tsx'
+import { HaokingMisc } from './Haoking.misc.tsx'
 
 export function HaokingPage() {
-  return (
-    <MantineProvider theme={theme}>
-      <Box
-        bg={theme.colors.black[4]}
-        h={rem('100vh')}
-        w={rem('100vw')}
-      >
-        <Flex
-          justify="flex-start"
-          direction="row"
-        >
-          <LeftAboutMe />
-          <RightAboutMe />
-        </Flex>
-      </Box>
-    </MantineProvider>
-  )
+  const [current, setCurrent] = useState('about me')
```

Comment:
I'm not a fan of having spaces in enums like this. Perhaps just "about"

Also you should likely use an enum for this. Search for TS Enums

Reply:
Stored pages as enum instead